### DISCUSSION TITLE
How to customize error.html

### DISCUSSION THREAD

**Discussion by devtrix** [sveltejs/kit#10593] [General]
For starters I feel there is a need for a fallback error component like mentioned in #8393.  In the mean time I need to style the error.html.  I am using Skeleton.dev, which uses Tailwindcss.  I found that directives like %sveltekit.assets%/ doesnt work.  Any suggestions are welcome.  Thanks in advance.

**Comment by git-no** [comment]
@devtrix 
Via Google search I found easily these helping descriptions :

https://kit.svelte.dev/docs/errors
https://learn.svelte.dev/tutorial/error-pages

`%sveltekit.assets%`did work on my test.

Without any code parts from you it will be nearly impossible to understand and to help what does not work on your code.
